adhocteam/Head-Start-TTADP
# [TTAHUB-330] Redo date picker select

## Description of change
The date range select on the dashboard page changed significantly from an earlier, previously unseen mockup. The inputs are spaced apart and function slightly differently, the checkmark behavior has changed, and instructions have been added.

## How to test
View the regional-dashboard. Use the region selector and the date range selector, ensure they still work and that the match the mockup in the JIRA ticket.

## Issue(s)
*https://ocio-jira.acf.hhs.gov/browse/TTAHUB-330

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
